autotest-agent/src/jvmMain/kotlin/com/epam/drill/agent/test/devtools/JsCoverageSender.kt

```diff
@@ -25,25 +25,27 @@ import com.epam.drill.agent.test.transport.TestAgentMessageSender
 import mu.KotlinLogging
 
 interface JsCoverageSender {
-    fun sendJsCoverage(testLaunchId: String): ResponseStatus?
+    fun sendJsCoverage(testLaunchId: String)
 }
 
 class JsCoverageSenderImpl : JsCoverageSender {
     private val logger = KotlinLogging.logger {}
 
-    override fun sendJsCoverage(testLaunchId: String) = DevToolStorage.get()?.run {
-        val coverage = takePreciseCoverage()
-        if (coverage.isBlank()) {
-            logger.trace { "coverage is blank" }
-            return null
+    override fun sendJsCoverage(testLaunchId: String) {
```

Comment:
Please, avoid committing formatting changes for unrelated code - you can partially unstage file to ensure that doesn't happen